yt_dlp/extractor/extractors.py

```diff
@@ -1973,7 +1973,6 @@
     VTXTVIE,
     WalyTVIE,
     ZattooIE,
-    ZattooLiveIE,
```

Comment:
Why the OldIEs are not added here?

Reply:
I'm certain that none of the resellers work anymore... but I have no opportunity to test it...
I can add them if you want?

Reply:
wdym "don't work"? Dont work in yt-dlp, or don't exist in the site anymore? If the old URLs don't exist, delete the extractor code altogether. Otherwise, don't remove the import

Reply:
By the time Zattoo changed its API to v3, the resellers still continued to use the v2 API.
That's the reason we have this `ZattooOldIE` around. But I guess that by now, all resellers have made the switch to the v3 API and the new URL scheme, but unfortunately I have no opportunity to test this...

There are indeed some resellers that don't seem to be reachable anymore. I will check and remove them.

Reply:
You still haven't deleted the old IE? Derive the other classes from the baseclass. Don't leave unused IEs

Reply:
I'm not quite sure if I understand you correctly here... Should I delete `ZattooOldIE` and `ZattooOldLiveIE` and then derive the resellers (e.g. `NetPlusIE`) directly from `ZattooBaseIE`?
I'm hesitant to do this because I don't know anything about the resellers for certain (URL scheme etc.) and have no way to test it... 

I could also remove all the resellers for now until someone can provide an account. How about that?

Reply:
This is what I meant: https://github.com/yt-dlp/yt-dlp/pull/2288/commits/91f437fe5722ecc7e3097c49e6c0aa48968c791a

Do you think this has any problems?

Reply:
That looks good to me, thanks!
I wouldn't bet that it works out of the box but this can only be checked after someone provides an account for a reseller.